to_do_item_class.rb

```diff
@@ -0,0 +1,13 @@
+class Item
+#items will know their name and if they are complete
+attr_reader :name, :status
+  def initialize(name)
+    @name = name
+    @status = false
+  end
+
+  def status
```

Comment:
This method, which is setting the status, will overwrite the attr_reader you have on this class. What were you hoping to do here? Maybe a different method name would help? Something like `complete`?